themes/shared/forms.scss

```diff
@@ -28,8 +28,10 @@
   border-top-color: #727272;
   border-bottom-color: #dadada;
   background: white url("/img/themes/fresh/textbox-bg.png") repeat-x top;
+  background-image: linear-gradient(to bottom, #b4b4b4 0px, #fff 2px);
```

Comment:
Since we going to use it for both themes, I'd suggest moving the image to `/img/themes/shared`.